pkg/lib/ldap/deps.go

```diff
@@ -0,0 +1,10 @@
+package ldap
+
+import (
+	"github.com/google/wire"
+)
+
+var DependencySet = wire.NewSet(
+	wire.Struct(new(Client), "*"),
```

Comment:
We don't need to list Client in the dependency set coz it will internally used by Service only